ui/core/sim.ts

```diff
@@ -243,6 +245,35 @@ export class Sim {
 				if (gearChanged) {
 					player.equipment = gear.asSpec();
 				}
+
+				// Include consumables in the player db
```

Comment:
Is there a better place to put this code, such as the same place where the items DB relevant to the player is created / updated?